mesonbuild/scripts/python_info.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
This is wrong, this script is NOT a python3 script, it is intentionally kept compatible with python2 and we are not going to change that now or in 20 years. :)

It has to run in python2 in order to support `find_installation('python2')`.

Reply:
hmm, on my Ventura 13.1 system, there is no `python` or `python2` installed by default/xcode

Reply:
i.e. in a clean shell, no homebrew:
```
 /usr/bin/env python
env: python: No such file or directory
```

Reply:
It gets run as `/path/to/detected/python/executable python_info.py` (which can be any python passed to find_installation, it need not be on PATH), the shebang is not actually used to run it. The point is that by convention, using "python" without a version is a sign to the reader that the script is expected to run on both.

Reply:
ahha, ok, no worries then